controllers/suggestionsController.js

```diff
@@ -15,12 +15,13 @@ class SuggestionsController {
 	}
 
 	/*
-	 This function chains 3 github API calls in order to create a pull request
-	 Right now this lacks error management (it assumes everything goes well from one call to another ... which will probably not be the case at the start)
+	 This function chains 4 github API calls in order to create a pull request
 	 @param accessToken: A valid accessToken to access the github API
 	 @param title: (string) Will be used as a title for the pull request creation
+	 @param description: (urlencoded JSON string) The state we want to record
```

Comment:
Use [JSdoc syntax](http://usejsdoc.org/tags-param.html#names-types-and-descriptions) for types.